wg-data-protection/charter.md

```diff
@@ -0,0 +1,76 @@
+# WG Data Protection Charter
+
+This charter adheres to the [wg-governance] guidance as well as
+the general conventions described in the [Kubernetes Charter README] and
+the Roles and Organization Management outlined in [sig-governance], where
+applicable to a Working Group.
+
+
+## Scope
+
+Data protection in Kubernetes context typically involves backup and recovery
+of two types of entities:
+* Kubernetes object resources
+* Persistent volume data
+We consider it a complicated and layered problem, including backup and recovery
+at persistent volume level, application level, and cluster level. Part of the
+working group’s charter is to define what Kubernetes native constructs are
+required to achieve these goals.
+
+Data Protection in Kubernetes includes backup and restore of applications
```

Comment:
Above it mention three levels. Here it only talks about application restore. Does it need to be mentioned separately?

Reply:
Hmm, it looks like I forgot to delete this line.  Good catch.